high-availability-endpoint/README.md

```diff
@@ -0,0 +1,53 @@
+high-availability-endpoint
+--------------------------
+
+This repository contains sample code for using the AWS Health API's high availability endpoint to determine which region to connect to in order to get the latest Health information.
+
+## Background
+
+AWS Health is a RESTful web service that uses HTTPS as a transport and JSON as a message serialization format. Your application code can make requests directly to the AWS Health API. When using the REST API directly, you must write the necessary code to sign and authenticate your requests. For more information, see the [AWS Health API Reference](https://alpha-docs-aws.amazon.com/health/latest/APIReference/).
```

Comment:
please replace with a GA doc link (not alpha)

Reply:
hi, 

Good catch!

I've changed the link, squashed the commit, and pushed the new commit as part of this PR.